meson.build

```diff
@@ -1871,6 +1871,26 @@ foreach target : target_dirs
                gui_app: exe['gui'])
     }
 
+    # Add qemu-system-cheri128 -> qemu-system-mips64cheri128 etc symlinks for
+    # backwards compatibility. Remove once the old names are unused.
+    # NB: This is a defective language with no string.replace, so we hack one
+    # up using split+join. https://github.com/mesonbuild/meson/issues/4007.
```

Comment:
That's quite disappointing... You'd think a new build system includes a simple string replace

Reply:
No string replace, no built-in symlink mechanism, it's all rather sad...